screens/Friends/index.js

```diff
@@ -0,0 +1,289 @@
+import React, { useEffect, useState } from 'react';
+import {
+  ActivityIndicator,
+  Alert,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  View,
+} from 'react-native';
+import { ListItem } from 'react-native-elements';
+import { useDispatch, useSelector } from 'react-redux';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import colors from 'consts/colors';
+import { PROFILE_SCREEN, USER_SEARCH_SCREEN } from 'screens/routes';
+import {
+  addFriend,
+  deleteFriendRequest,
+  getUsersFriendRequests,
+} from 'functions/friends';
+import { updateUserFriends } from 'store/reducers/user';
+import errorMessages from 'consts/errorMessages';
+
+const Friends = ({ navigation }) => {
+  const user = useSelector((state) => state.user.user);
+  const [friendRequests, setFriendRequests] = useState([]);
+  const [handlingFriendRequest, setHandlingFriendRequest] = useState(false);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    fetchRequests();
+  }, [dispatch]);
```

Comment:
why is dispatch set as dep? wouldn't it be like friendRequests?

Reply:
oh yeah not necessary, probably was using dispatch in useEffect at one point and didnt remove